vectorstores/alloydb/vectorestores.go

```diff
@@ -115,7 +114,7 @@ func (vs *VectorStore) AddDocuments(ctx context.Context, docs []schema.Document,
 			if val, ok := metadata[metadataColumn]; ok {
 				valuesStmt += fmt.Sprintf(", $%d", len(values)+1)
 				values = append(values, val)
-				delete(metadata, metadataColumn)
+				// delete(metadata, metadataColumn)
```

Comment:
Why is this commented out?

Reply:
I forgot to delete that line, since I asked the question the other day about having stored metadata columns separately and metadata Json along in the same table, I need not to delete the field from the metadata map for it to be stored in metadataJson as well, which if down below this for statement!

Reply:
I forgot to delete that line, since I asked the question the other day about having stored metadata columns separately and metadata Json along in the same table, I need not to delete the field from the metadata map for it to be stored in metadataJson as well, which if down below this for statement!